text/conversations/04_defiance_bay_brackenbury/04_cv_bellasege.stringtable

```diff
@@ -565,7 +565,7 @@ Sie seufzt. "Ich fürchte, ich werde mit leeren Händen nach Revua zurückkehren
     </Entry>
     <Entry>
       <ID>110</ID>
-      <DefaultText />
+      <DefaultText />Bank Node 110</DefaultText>
```

Comment:
Man kann die ganzen Bank Notes auch wieder entfernen, die haben ja eh keinen Inhalt. Ich dachte nur mal, dass ich die deutsche Version mit der englischen synche, falls sich da in Zukunft mal was tut...